src/lib/encoder.ts

```diff
@@ -80,6 +81,55 @@ const decodeDataSourceWithHash = (value: string): URLDataWithHash => {
   return { hashFunction: hashFunction.name, hash: dataHash, url: dataSource };
 };
 
+const encodeCompactBytesArray = (values: string[]): string => {
+  let compactBytesArray = '0x';
+
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] === '')
```

Comment:
Doesn't the next `isHex` cover this anyway?

Reply:
`isHex('')` actually returns `true`